### PR TITLE
fix: [SRM-13682]: Removed no runtime inputs available image during run pipeline

### PR DESCRIPTION
**Pull request by praneshg239**
#### Screenshots

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by karansaraswat19 — APPROVED** [review]

File: src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/TemplatisedRunTimeMonitoredService.tsx [hunk 1]
```diff
@@ -104,46 +103,44 @@ export default function TemplatisedRunTimeMonitoredService(
...
-                      enableConfigureOptions={false}
-                    />
-                  )
-                } else {
-                  return (
-                    <>
-                      <FormInput.MultiTextInput
-                        key={input.name}
+            {runtimeInputs.length
+              ? runtimeInputs.map(input => {
```
Comment (harshilbits) [inline src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/TemplatisedRunTimeMonitoredService.tsx:107]: Run time screen is visible at multiples places. Can you please check , we might have to remove everywhere.
1. Run pipeline form.
2. In verify step.
3. While creating monitored service from monitored service teplate.

Comment (praneshg239) [reply src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/TemplatisedRunTimeMonitoredService.tsx:107]: sure, will check

Comment (praneshg239) [reply src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/TemplatisedRunTimeMonitoredService.tsx:107]: Just checked, it needs to be removed from all the places

Comment (harshilbits) [reply src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/TemplatisedRunTimeMonitoredService.tsx:107]: Current change is about removing from one place only. Please di the changes at other places also.

Comment (praneshg239) [reply src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/TemplatisedRunTimeMonitoredService.tsx:107]: I thought it is a common component, let me check

**Review by harshilbits — DISMISSED** [review]

File: src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/TemplatisedRunTimeMonitoredService.tsx (hunk 1 above)
Comment (praneshg239) [reply src/modules/85-cv/components/PipelineSteps/ContinousVerification/components/ContinousVerificationInputSetStep/components/TemplatisedRunTimeMonitoredService/TemplatisedRunTimeMonitoredService.tsx:107]: Done

**Review by harshilbits — APPROVED** [review]

**Review by Nyctophiliac1918 — APPROVED** [review]

**Review by deepesh-ui — APPROVED** [review]

**Pull request merged**